products/bash/configure-apic-v10.sh

```diff
@@ -14,7 +14,7 @@
 #
 # PARAMETERS:
 #   -n : <namespace> (string), Defaults to "cp4i"
-#   -r : <release-name> (string), Defaults to "demo"
+#   -r : <release-name> (string), Defaults to "apic-demo"
```

Comment:
Let's use a shorter name due to APIC requiring project name + release name <= 14 characters.

```suggestion
#   -r : <release-name> (string), Defaults to "ademo"
```